### DISCUSSION TITLE
Is there are way to keep Freeplane active even if there is no currently open file?

### DISCUSSION THREAD

**Discussion by peterstev** [freeplane/freeplane#1540] [Help] [answered]
On Mac, when I close the last file, the program exits. Is there a way to prevent this? 

I find it difficult to switch back to the app (Alt-tab doesn't find it), especially when I want to create a new map. Also, I don't like to keep maps open that I am not working on to reduce the chance of modifying them accidentally.

Thanks in advance!

Peter

**Comment by i-plasm** [comment, marked as answer]
Hi, have you tried closing the last map with `  Command + W ` ? It should just close just the map, not Freeplane, (it is left open and you can find it by window cycling with `  Command + Tab `, at least on the macs I've tried.

**Comment by peterstev** [comment]
Thank you i-plasm, that did the trick. It works on both Mac versions (mac and intel hardware). It is strange, you can close all but the last tab in the GUI. When there is only one file open, the tab bar disappears, and you can only close the window with the mouse. Closing the window closes the program (it is equivalent to Command + Q, not Command + W on all tabs).

**Comment by i-plasm** [comment]
Glad it helped. I forgot to mention, in the ` Edit -> Viewing ` option all the open maps enter non-editable mode, which could help your concerns as well.  ` Edit -> Editing`  goes back to editing mode. (***Edit: it is not too obvious from the UI when non-editable mode is on; I've almost lost notes when I've typed on the note panel thinking I was truly editing the note. Use with caution***)  Re: the close behaviour on Macs: Actually one can emulate the Mac close behaviour with a small groovy script. In essence what the script does is 1) Requests a "close all open maps" action - the user will be prompted to save changes if necessary -, and 2) Perform a minimization (Windows) or a "Send to back" action (Mac). You can read more info on the script header. The script is [found here](https://raw.githubusercontent.com/i-plasm/freeplane-scripts/main/src/scripts/maclikeClose.groovy).
